Disable bandit-baseline command. It depends on GitPython module that is not
part of Solaris now.

diff -ruN bandit-1.6.2.orig/setup.cfg bandit-1.6.2/setup.cfg
--- bandit-1.7.0/setup.cfg
+++ bandit-1.7.0/setup.cfg
@@ -29,7 +29,6 @@ project_urls =
 console_scripts = 
 	bandit = bandit.cli.main:main
 	bandit-config-generator = bandit.cli.config_generator:main
-	bandit-baseline = bandit.cli.baseline:main
 bandit.blacklists = 
 	calls = bandit.blacklists.calls:gen_blacklist
 	imports = bandit.blacklists.imports:gen_blacklist
